src/sql/workbench/contrib/query/browser/actions.ts

```diff
@@ -67,6 +67,12 @@ export class SaveResultAction extends Action {
 	}
 
 	public async run(context: IGridActionContext): Promise<boolean> {
+		this.notificationService.prompt(Severity.Info, localize('jsonEncodingPrompt', "Results encoding will not be saved when exporting to JSON, remember to save with desired encoding once file is created."),
```

Comment:
Does this really need to be a prompt? Adding in clicks for users isn't usually desirable - seems like this could be just a info toast instead (and then allow users to never get notified about that again)

Reply:
I'll change it to a toast, just want the user to know about it.